Documentation/CRDs/Cluster/network-providers.md

```diff
@@ -515,7 +532,8 @@ If any CephClusters have Multus enabled (`network.provider: "multus"`), follow t
 steps above before continuing.
 
 **Step 2**
-Begin by setting `CSI_DISABLE_HOLDER_PODS: "true"` (and `CSI_ENABLE_HOST_NETWORK: "true"` if desired).
+Begin by setting `CSI_DISABLE_HOLDER_PODS: "true"` -- also set the desired value of
+`CSI_ENABLE_HOST_NETWORK` if needed.
```

Comment:
Again, isn't `CSI_ENABLE_HOST_NETWORK: true` required when holder pods are disabled? Or how do they know if it's needed?

Reply:
They can read the networking section on CSI host networking to decide if they want to enable it or not. It's not required. But we did talk about making another update after this PR to make that situation unsupported.